### ISSUE TITLE
In preview mode make layer number label visible when print settings is expanded 

### ISSUE THREAD

**Issue by oishii-code** [Ultimaker/Cura#6446]
**Is your feature request related to a problem? Please describe.**
I always have the print settings expanded so when I'm previewing the layers using the slider, the layer number label is always hidden behind it.

**Describe the solution you'd like**
Move the layer number label to be above or below the layer preview slider so that it is always visible, even when the print settings is expanded.

**Describe alternatives you've considered**
- Moving the print settings to the left, however I think this would waste space that could be better used to view the model.
- Move the layer slider somewhere else.

**Affected users and/or printers**
Any users who never collapses the print settings.

**Additional context**
Cura v4.3.0

![image](https://user-images.githubusercontent.com/25396158/65826326-9efd7480-e237-11e9-9326-9d3fab2d12bc.png)

![image](https://user-images.githubusercontent.com/25396158/65826328-c0f6f700-e237-11e9-8b3e-3fa6d79c81fb.png)

**Comment by oishii-code** [comment]
Closed. I noticed the printer settings windows could be moved.

**Comment by oishii-code** [comment]
I added some more info and reopened it in case anyone has any feedback.

**Comment by Ghostkeeper** [comment]
Yeah I've seen this before (but can't find the original bug report). We do consider this a bug. It's a bit hairy to fix because it involves space claims by plug-ins.
